generator/config/query-operators.yaml

```diff
@@ -0,0 +1,9 @@
+- name: and
+  args:
+    - name: query
+      type: resolvesToQueryOperator
+      isVariadic: true
```

Comment:
We should throw an error if the number of query objects is lower that 1. In Doctrine, they use a required argument before the variadic one.

https://github.com/doctrine/mongodb-odm/blob/9b7ed61f4e12ac0f4f8a18f65dfcfd8a05066dab/lib/Doctrine/ODM/MongoDB/Aggregation/Expr.php#L162